firstvoices/backend/tasks/visibility_tasks.py

```diff
@@ -0,0 +1,71 @@
+from celery import shared_task
+from django.db import transaction
+
+from backend.models import DictionaryEntry, SitePage, Song, Story, StoryPage
+from backend.models.jobs import BulkVisibilityJob, JobStatus
+from backend.models.sites import SiteFeature
+from backend.models.widget import SiteWidget
+from backend.search.tasks.site_content_indexing_tasks import (
+    sync_all_site_content_in_indexes,
+)
+
+
+@shared_task
+def bulk_visibility_change_job(job_instance_id):
```

Comment:
Functions should have verb names, so let's call this something like start_visibility_job or bulk_change_visibility